src/modules/functionality.js

```diff
@@ -0,0 +1,77 @@
+import addObjToLocalStorage from './objectToLS.js';
+import { store, taskArray } from './store.js';
+
+const submit = document.querySelector('.fa-right-from-bracket');
+const tasksContainer = document.querySelector('.toDo-list');
+const taskName = document.querySelector('.new-task');
+
+const addTask = (e) => {
+  e.preventDefault();
+  if (e.target.classList.contains('fa-right-from-bracket')) {
+    if (!taskName.value) return;
+    addObjToLocalStorage(taskName.value);
+    taskName.value = '';
+  }
+};
+
+const deleteTask = (e) => {
+  if (e.target.classList.contains('delete-icon')) {
+    const btn = e.target;
+    const removableTask = btn.closest('li');
+    const taskIndex = removableTask.dataset.indexNumber;
+
+    // Remove element from array
+    taskArray.splice(taskIndex - 1, 1);
+    for (let i = 0; i < taskArray.length; i += 1) {
+      taskArray[i].index = i;
+    }
+
+    localStorage.setItem('taskInput', taskArray);
+    store();
+  }
+};
+
+const edit = (e) => {
```

Comment:
You should use a more descriptive function name for code readability. Maybe you can rename it to `editTask` instead of `edit`.